inb4404.py

```diff
@@ -31,10 +32,33 @@ def main():
     else:
         download_from_file(thread)
 
+    if args.title:
+        try:
+            import bs4
```

Comment:
any reason why you import the whole bs4 module here instead of `from bs4 import BeautifulSoup` like on line 49?

Reply:
To keep the line short.
If the import should fail, we'd catch it both ways, so I went with the short version.

But if the import should succeed, the module is out of the scope of `get_title_list()`, and we need to add it again over there.